Apps/phcrxcavator/exclude_files.txt

```diff
@@ -0,0 +1 @@
+.git*
```

Comment:
what's the reason for ignoring all `.git` files? the `.git` directory is excluded by default and usually `.gitignore` and other such files are committed to the repo